CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog for the Mapbox Search SDK for Android
 
+## 2.0.0
+
+### Breaking changes
+- [Address Autofill] Coordinates aren't retrieved speculatively anymore
```

Comment:
I'm not sure if "speculatively" is suitable in this context.

I'd say something like
```
[Address Autofill] `AddressAutofill` is a two-steps search now, i.e. it returns `AddressAutofillSuggestion`s (without geo coordinate) at the first step and `AddressAutofillResult` after suggestion selection.
```

The same is applicable for Place Autocomplete.